.github/workflows/ci.yml

```diff
@@ -18,4 +18,7 @@ jobs:
     uses: eclipse-platform/eclipse.platform.releng.aggregator/.github/workflows/checkMergeCommits.yml@master
   build:
     uses: eclipse-platform/eclipse.platform.releng.aggregator/.github/workflows/mavenBuild.yml@master
-
+  public-test-results:
+    if: ${{ always() }}
```

Comment:
I wonder if the test publication can/should run always or only if the `build` job succeeded?
If the later is true, then this if is obsolete.

Reply:
No it must run always because the build can fail with a failing test and we want to cover this case..